components/sidebar/index.js

```diff
@@ -45,6 +45,13 @@ function mapStateToProps(state) {
         config.ExperimentalGroupUnreadChannels === GroupUnreadChannels.DEFAULT_ON
     );
 
+    const channelSwitcherOption = getBoolPreference(
+        state,
+        Preferences.CATEGORY_SIDEBAR_SETTINGS,
+        'channel_switcher_section',
+        'true'
```

Comment:
Last argument actually requires boolean value (although string works the same as well)

Reply:
will make the changes

Reply:
will not!